src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -178,7 +178,7 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
 
         public override void Shutdown()
         {
-            Dispose();
+            // Do nothing for now
```

Comment:
already cleaned up on DocumentationBrowserViewExtension::Dispose method.

Reply:
And where is that called from now?

Reply:
I always forget the deal here...
between view extensions and extensions we have different behavior regarding disposal and shutdown being called...

This would be very nice to cleanup and unify for 3.0...

Reply:
Called from the ViewExtensionManager::Dispose which is called when Closing the main DynamoView
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Extensions/ViewExtensionManager.cs#L252